Homework/homework.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Document</title>
+	</head>
+	<body>
+		<script>
+			// Exercise 1: Write a program that ask the user for his name and prints a greeting.
+			//let name = prompt("Moin, how is your Name?");
+			//document.write("Hello " + name + ", welcome to the Learn to Code course.");
+
+			// Exercie 2: Choose a simple Ascii art from the website https://www.asciiart.eu/. Output it to the console.
+			// console.log("                     ______                    " + "<br>");
```

Comment:
There is no need for <br> in a console.log.
console.log does not support HTML and every console.log is already in it's own line.